src/app/resources/resources.component.html

```diff
@@ -26,6 +26,7 @@
     <ng-container *ngIf="isAuthorized">
       <button *ngIf="!parent" mat-mini-fab routerLink="add"><mat-icon>add</mat-icon></button>
       <planet-filtered-amount [table]="resources"></planet-filtered-amount>
+      <planet-tag-selected-input [selectedIds]="tag.value" [parent]="parent"></planet-tag-selected-input>
```

Comment:
I realize you don't actually need the `#tag` for this, you can just set `selectedIds` to `tagFilter.value`

Also, we're running out of horizontal space in this toolbar.  Can you put this text below the "Showing X of Y resource(s)"?

Reply:
Thanks for a good advice @paulbert.